### PR TITLE
[WIP][StyleCopy warnings]Added request functionality

### PR DESCRIPTION
**Pull request by buggy213**
As described in #1075, currently with no support in the trade GUI

### CODE REVIEW

File: Assets/Scripts/Controllers/TradersController.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			}
+			
+			traders.Add (t.Name, t);
+			
+		}
+		
+		// Gets a random trader from the list
+		public Trader RequestRandomTrader () {
+			
+			Trader joe = traders.Values.ToList()[UnityEngine.Random.Range (0, traders.Count)];
```
Comment (sboigelot) [inline Assets/Scripts/Controllers/TradersController.cs]: UnityEngine.Random.Range (0, traders.Count)]

should be

UnityEngine.Random.Range (0, traders.Count - 1)]

I think

File: Assets/Scripts/Controllers/TradersController.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+namespace Controllers
+{
+	/// <summary>
+	/// Saves trader data for trader persistence (traders return time and time again to rip you off)
+	/// </summary>
+	public class TradersController
```
Comment (zwrawr) [inline Assets/Scripts/Controllers/TradersController.cs]: Tabs , need replacing with 4 spaces

File: Assets/Scripts/Controllers/TradersController.cs (hunk 1 above)
Comment (UnknownSoldier1) [inline Assets/Scripts/Controllers/TradersController.cs]: No, the int version of Random.Range goes from min (inclusive) to max (exclusive). That code is correct.

Comment (sboigelot) [inline Assets/Scripts/Controllers/TradersController.cs]: ok

> Returns a random integer number between min [inclusive] and max [exclusive] (Read Only).

this one always confuse me, sorry.

**Comment by Grenkin1988** [comment]
How to run StyleCop : https://github.com/TeamPorcupine/ProjectPorcupine/wiki/StyleCop

**Comment by buggy213** [comment]
oops

**Pull request closed without merging**